software-templates/techdocs-template/skeleton/README.md

```diff
@@ -0,0 +1,63 @@
+[![Package Status](https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/actions/workflows/publish.yaml/badge.svg)](https://${{values.destination.host}}/${{values.destination.owner}}/${{values.destination.repo}}/actions/workflows/publish.yaml)
+
+
+# ${{values.name}}
+
+${{values.description}}
+
+## About
+
+This is a default [README.md](README.md) file generated by the template. Customize it for your project.
+
+Refer to the [index.md](docs/index.md) file for instructions on how to add documentation.
```

Comment:
should we hard-link this to something in one of our repos so it can stay up to date as we change it over time?

Reply:
@jenreiher-bcgov - I could link it to the [devhub-template skeleton index file](https://github.com/bcgov/devhub-templates/blob/main/software-templates/techdocs-template/skeleton/docs/index.md) instead. @sheaphillips your thoughts on this?